cmd/status.go

```diff
@@ -551,50 +562,94 @@ func displayFailedEntries(client *server.Client, given *set.Set) {
 
 // displayAllEntries prints out details about all entries in the given
 // set.
-func displayAllEntries(client *server.Client, given *set.Set) {
+func displayAllEntries(client *server.Client, given *set.Set, showRemotePaths bool, transformer put.PathTransformer) {
 	all, err := client.GetFiles(given.ID())
 	if err != nil {
 		die(err.Error())
 	}
 
-	displayEntries(all)
+	displayEntries(all, showRemotePaths, transformer)
 }
 
 // displayEntries prints info about the given file entries to STDOUT.
-func displayEntries(entries []*set.Entry) {
+func displayEntries(entries []*set.Entry, showRemotePaths bool, transformer put.PathTransformer) {
 	if len(entries) == 0 {
 		return
 	}
 
-	printEntriesHeader()
+	displayHeader(showRemotePaths)
+
+	warnedAboutTransformer := false
 
 	for _, entry := range entries {
-		var date string
+		var remotePath string
 
-		if entry.LastAttempt.IsZero() {
-			date = "-"
-		} else {
-			date = entry.LastAttempt.Format(dateShort)
+		if showRemotePaths {
+			remotePath, warnedAboutTransformer = getRemotePath(entry.Path, transformer, warnedAboutTransformer)
 		}
 
-		cols := []string{
-			entry.Path,
-			entry.Status.String(),
-			humanize.IBytes(entry.Size),
-			fmt.Sprintf("%d", entry.Attempts),
-			date,
-			entry.LastError,
-		}
+		displayEntry(entry, remotePath)
+	}
+}
+
+// displayHeader adds a column for remote path if showRemotePath is true and
+// prints the header.
+func displayHeader(showRemotePath bool) {
+	cols := []string{"Path", "Status", "Size", "Attempts", "Date", "Error"}
 
-		cliPrintRaw(strings.Join(cols, "\t"))
-		cliPrintRaw("\n")
+	if showRemotePath {
+		cols = append(cols, "Remote Path")
```

Comment:
It would work better if it was the second column after Path, and if Path was renamed "Local Path" for clarity.